app.py

```diff
@@ -8,7 +8,9 @@
 from openai import AssistantEventHandler
 from tools import TOOL_MAP
 from typing_extensions import override
+from dotenv import load_dotenv
```

Comment:
there will be a merge conflict right here with the PR: https://github.com/ryo-ma/gpt-assistants-api-ui/pull/15 I trust you know how to fix.

although I could send a consolidated PR from my forked `main` branch (that I merged the 2 development branches), I decided to split the PRs and leave it up to you.